applications/endoscopy_tool_tracking/cpp/main.cpp

```diff
@@ -151,16 +160,19 @@ class App : public holoscan::Application {
             "pool", 1, lstm_inferer_block_size, lstm_inferer_num_blocks),
         Arg("cuda_stream_pool") = cuda_stream_pool);
 
-    const uint64_t tool_tracking_postprocessor_block_size = 107 * 60 * 7 * 4;
-    const uint64_t tool_tracking_postprocessor_num_blocks = 2;
+    // the tool tracking post process outputs
+    // - a RGBA float32 color mask
+    // - coordinates with x,y and size in float32
+    const uint64_t tool_tracking_postprocessor_block_size =
```

Comment:
Is leaving it as `BlockMemoryPool` instead of `RMMAllocator` here is mainly so we don't have to assign a different allocator depending on the version of Holoscan? (or was there some performance benefit to keeping the block memory pool instead?)

Reply:
This is just to make the code less cluttered and easier to read. For the video replayer we only have RMM or Unbound allocator, so adding the extra code to use RMM allocator makes sense. But for the others I think using BlockMemory and RMM allocator makes no difference.

Reply:
thanks for confirming. I am fine with this as-is then.